src/components/panels/analysis/ScoreBubble.tsx

```diff
@@ -66,22 +66,31 @@ function ScoreBubble({
         textAlign: "center",
         padding: "0.15rem",
         borderRadius: theme.radius.sm,
-        width: size === "md" ? "4rem" : "3.5rem",
+        width: size === "md" ? "5.0rem" : "3.5rem",
```

Comment:
Are these changes from 4rem to 5rem really necessary? I think it would be better to change it to minWidth and keep it at 4rem